common/src/main/webapp/less/style.less

```diff
@@ -338,6 +338,10 @@ tfoot {
   margin-top: 0;
   cursor: pointer;
   min-height: 36px;
+  flex-shrink: 2;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
```

Comment:
These 3 properties seem like something we want to use in a number of places. Do we want a mixin?

Reply:
Yes, I misread. I though we were on sass here.